datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedPartitionAssigner.java

```diff
@@ -137,13 +140,15 @@ public Map<String, Set<DatastreamTask>> assignPartitions(
       }
     }
 
+    LOG.info("Sorting recognized partitions on byte rate");
```

Comment:
any other metadata needed to know what datastream this log belongs too? by looking into this log line one might not have enough information to interpret what this is for. Or the above log which has datastream name is sufficient for us? https://github.com/linkedin/brooklin/pull/951/files#diff-430d9ed3200fdf27009cd18273178da8a789cdec2e410b90ac5bb00562704d69R88

Reply:
Datastream name in the START and END messages is sufficient. No need to repeat it in every message printed.